hr_employee_firstname/models/hr_employee.py

```diff
@@ -92,14 +150,6 @@ def _install_employee_firstname(self):
         records._inverse_name()
         _logger.info("%d employees updated installing module.", len(records))
 
-    def _update_partner_firstname(self):
```

Comment:
Here you are removing part of the functionality of the module. You could keep this logic by checking whether the contact_firstname module is installed. (Please avoid yet another glue module...)